### ISSUE TITLE
Free energy calculation

### ISSUE THREAD

**Issue by Liuly98** [shankar1729/jdftx#117]
Dear Shankar,
When I followed the jdftx tutorial of "Free energy: Vibrations" on the website,I had a small confusion about the formula of the correct binding free energy for H2 at room temperature, which is "E_H2 + Avib_H2 + (3/2)kT - 2(EH + (3/2)kT)". 
This formula has indeed considered the contributions of vibrational free energy(Avib_H2) and translational free energy(3/2kT) to H2 free energy, but it seemed to ignore the rotational free energy of H2, right?  
For H2, its classical rotational energy is KT according to the Equipartition theorem of energy, which is close to the translational energy(3/2kT) , so I think the rotational energy should not be ignored if we have considered  the translational energy. Was my thought above reasonable?
By the way, I also felt confused about the translational "free" energy(3/2kT), because in my opinion it (ie.3/2kT) should be called translational energy(E(trans)) according to the Equipartition theorem, and translational "free" energy can be written as "E(trans)-TS(trans)" just like "Avib = Evib -TSvib".
I  am looking forward to any of your response and help!
Thanks a lot !
Best regards !

**Comment by shankar1729** [comment]
You're right: the rotational contribution should also be included as it is of the same order.

Also, it should be atomization energy / binding energy as the entropy is not included. One could also include the translational and rotational entropy to get the full free energy, but that would be a bit more complicated.

If you would like to draft edited numbers with these changes for energy and/or free energy, that would be great and I'd be happy to update the tutorial accordingly.

Thanks!
Best,
Shankar

**Comment by Liuly98** [comment]
Thanks for your response! I will try it later.
Best regards!
